network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -777,15 +779,15 @@ public void updateNetworks(List<Resource<NetworkAttributes>> resources) {
         for (List<Resource<NetworkAttributes>> subresources : Lists.partition(resources, BATCH_SIZE)) {
             BatchStatement batch = new BatchStatement(BatchStatement.Type.UNLOGGED);
             for (Resource<NetworkAttributes> resource : subresources) {
-                batch.add(psUpdateNetwork.bind(
+                batch.add(creationalStatement(psUpdateNetwork.bind(
```

Comment:
creationalStatement method name could be modified since it is now used with update statements.

Reply:
Done